### PR TITLE
docs(provider): readme update for Stripe Billing Package Integration

### PR DESCRIPTION
**Pull request by Tyagi-Sunny**
## Description
readme update for stripe billing package integration

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by yeshamavani** [comment]
PR title - this is not a feature just docs are updated

File: README.md [hunk 1]
```diff
@@ -93,21 +91,69 @@ The loopback4-billing package relies on the configuration of the chosen billing
 
 ### Step 1: Set Up Billing Provider
 
-#### ChargeBee -
-
+#### For ChargeBee -
 To use Chargebee as the billing provider, you need to configure the Chargebee API keys and site URL in your application. You can set these values in the environment variables of your LoopBack 4 project.
 
 ```
 CHARGEBEE_API_KEY=your_chargebee_api_key
```
Comment (a-ganguly) [inline README.md:98]: What is the use of this variable.. I do not see this variable used below any where.

File: README.md [hunk 2]
```diff
@@ -93,21 +91,69 @@ The loopback4-billing package relies on the configuration of the chosen billing
...
 CHARGEBEE_SITE_URL=your_chargebee_site_url
```
Comment (a-ganguly) [inline README.md:99]: Same for this, no use of this declared variable.

**Pull request merged**